### DISCUSSION TITLE
send more data via mqtt (all strings additionally)

### DISCUSSION THREAD

**Discussion by ronn9** [tbnobody/OpenDTU#162] [Ideas]
Is it possible to send more data via mqtt?
Add the individual strings.

**Comment by tbnobody** [comment]
All individual strings are already published via MQTT. 

- Channel 0: AC
- Channel >0: DC Strings

**Reply by ronn9** [reply]
Sorry. I didn't see it at first glance.

**Comment by ronn9** [comment]
can be ignored since it already exists. Sorry
